packages/paste-core/components/chat-composer/stories/index.stories.tsx

```diff
@@ -0,0 +1,320 @@
+import * as React from 'react';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {AttachIcon} from '@twilio-paste/icons/esm/AttachIcon';
+import {SendIcon} from '@twilio-paste/icons/esm/SendIcon';
+import type {EditorState} from '@twilio-paste/lexical-library';
+import {$getRoot, $createParagraphNode, $createTextNode} from '@twilio-paste/lexical-library';
+import {
+  MinimizableDialog,
+  MinimizableDialogButton,
+  MinimizableDialogContainer,
+  MinimizableDialogHeader,
+  MinimizableDialogContent,
+} from '@twilio-paste/minimizable-dialog';
+import {ChatIcon} from '@twilio-paste/icons/esm/ChatIcon';
+import {DownloadIcon} from '@twilio-paste/icons/esm/DownloadIcon';
+import {
+  ChatLog,
+  ChatMessage,
+  ChatMessageMeta,
+  ChatMessageMetaItem,
+  ChatBubble,
+  ChatAttachment,
+  ChatEvent,
+  ChatAttachmentDescription,
+  ChatAttachmentLink,
+  ChatBookend,
+  ChatBookendItem,
+  ComposerAttachmentCard,
+} from '@twilio-paste/chat-log';
+import {Avatar} from '@twilio-paste/avatar';
+import type {Story} from '@storybook/react';
+
+import {ChatComposer} from '../src';
+import type {ChatComposerProps} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Chat Composer',
+  component: ChatComposer,
+};
+
+const defaultConfig: ChatComposerProps['config'] = {
+  namespace: 'foo',
+  onError: (error: Error) => {
+    throw error;
+  },
+};
+
+const initialText = (): void => {
+  const root = $getRoot();
+
+  if (root.getFirstChild() === null) {
+    const paragraph = $createParagraphNode();
+    paragraph.append($createTextNode('Hello'));
+
+    root.append(paragraph);
+  }
+};
+
+const tallInitialText = (): void => {
+  const root = $getRoot();
+
+  if (root.getFirstChild() === null) {
+    for (let i = 0; i < 10; i++) {
+      const paragraph = $createParagraphNode();
+      paragraph.append($createTextNode());
+      paragraph.append($createTextNode('this is a really really long initial value'));
+
+      root.append(paragraph);
+    }
+  }
+};
```

Comment:
question (non-blocking): should we abstract this away into a helper function exported from the ChatComposer file?

```js
setInitialText('hello')
// Does the above behind the scenes
// can split on `\n` to turn a string into paragraphs, or accept an array of strings alternatively
```

Reply:
I made a version of this function as a helper for the `initialText` prop: https://github.com/twilio-labs/paste/blob/859978378eb1e1e4d4f3609cea7dedea508ab948/packages/paste-core/components/chat-composer/src/helpers.ts#L3

It doesn't support the new lines... Maybe we could add it later if people want it?